src/libslic3r/PrintConfig.cpp

```diff
@@ -4327,15 +4340,6 @@ def = this->add("filament_loading_speed", coFloats);
         default: assert(false);
         }
     }
-
-    def = this->add("detect_narrow_internal_solid_infill", coBool);
```

Comment:
Can we add this option to the ignore list in handle_legacy() function too? so that existing project file won't throw errors

Reply:
Done